examples/common-workload.yaml

```diff
@@ -35,7 +35,7 @@ spec:
         ports:
         - name: http
           containerPort: 80
-          hostPort: 8081
+          hostPort: 8094
```

Comment:
The kube `hostPort` stays 8081.

Reply:
cool - was just going to ask you about this.  thank you